joblib/test/test_pickle.py

```diff
@@ -0,0 +1,21 @@
+"""
+Test that pickle and dill output the same bytes
+"""
+import pickle
+
+import nose
+
+from joblib.test.common import dill, with_dill, with_py3
+
+
+def global_func():
+    pass
+
+
+@with_py3
+@with_dill
+def test_dumps_eq():
+    # Test that dill and pickle produce the same results
```

Comment:
Is this still necessary if dill replaces pickle completely? For my use case I can just clear the cache after the dill update.